sklearn/linear_model/coordinate_descent.py

```diff
@@ -1063,6 +1063,8 @@ def fit(self, X, y):
 
         if hasattr(self, 'l1_ratio'):
             model_str = 'ElasticNet'
+            if self.l1_ratio == 0:
+                raise ValueError("l1_ratio = 0 not supported")
```

Comment:
Oops: passing an `np.array` as the `l1_ratio` does not work with the addition of my check.  Sorry, this check got hairy,,,